UnityProject/Assets/Scripts/Systems/Crafting/Ingredient.cs

```diff
@@ -1,26 +1,26 @@
 ﻿using System;
 
 
```

Comment:
add a namespace, it this case it would be ``Systems.Crafting``

Reply:
Yeah this code is from the old crafting system. I shouldn't commit these changes. But ok, I'll add a namespace

Reply:
it would be best if we merged Crafting and CraftingV2. Why did you need to make a separate thing?

Reply:
The systems have many differences, and also there are about a hundred or more recipes running on the old crafting system. I thought it could be better to move the old recipes onto the new crafting system gradually.

I noticed that we did so, iirc, with the new door system and ItemAttributesV2

Reply:
Are you sure you want me to add namespaces for the old crafting system which might be removed?

![image](https://user-images.githubusercontent.com/42589308/127833577-8872f22f-85a1-4099-8381-22014facf438.png)

Reply:
no, it is fine then.